packages/quantic/force-app/main/default/lwc/quanticQueryError/quanticQueryError.js

```diff
@@ -0,0 +1,108 @@
+import { LightningElement, api, track } from 'lwc';
+import {registerComponentForInit, initializeWithHeadless} from 'c/quanticHeadlessLoader';
+
+let disconnectedException = 'Disconnected';
```

Comment:
IMO you should not declare constants outside a component like that. In fact, now that localization is merged, those strings should go into the translation file.

Reply:
Please move these constants in the translation file.

Reply:
Those value came from `state` (in fact, they are the values that `type` can take ) no matter the language there will be the same values

Reply:
![image](https://user-images.githubusercontent.com/85293211/127874660-38306394-0b17-4796-aaa4-6500acf452ca.png)
 the modification that I should do is the traduction of `errorTitle`